source/_docs/start-state.md

```diff
@@ -22,7 +22,7 @@ We base our repository on the canonical source from drupal.org, and then extend
 - [Drupal 8](https://github.com/pantheon-systems/drops-8) <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="Install Requires SFTP Mode"><em class="fa fa-info-circle"></em></a>
 - [Drupal 7](https://github.com/pantheon-systems/drops-7)
 
-## Start with a Distribution
+## Start with a Custom Upstream for Public Distributions
```

Comment:
These are not custom but are the pantheon-supported (for now) upstreams. I think we can leave this as 'Distribution' for now and revisit how we define this class of upstream during the Public Upstreams project